src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.container.js

```diff
@@ -86,6 +84,7 @@ const mapDispatchToProps = (dispatch: ReduxDispatch): DispatchersFromRedux => ({
         dispatch(actionCreator(searchPageActionTypes.CURRENT_SEARCH_INFO_SAVE)({ searchScopeType, searchScopeId, formId, currentSearchTerms }));
     },
     addFormIdToReduxStore: (formId) => { dispatch(addFormData(formId)); },
+    removeFormDataFromReduxStore: (formId) => { dispatch(addFormData(formId)); },
```

Comment:
Don't you think we can completely remove the relevant object properties from the redux store? (using the removeFormData action you've created)

Reply:
I think there was a bug. I believe now the key is deleted completely :) 

Check that the keys from the object are removed when the form 'unmounts'
![image](https://user-images.githubusercontent.com/4181674/92927936-0e6b7480-f447-11ea-9811-e370c89d5329.png)
![image](https://user-images.githubusercontent.com/4181674/92927954-188d7300-f447-11ea-835c-1a68dc9010cf.png)